test/aggregators/stacks-address.test.ts

```diff
@@ -1,6 +1,14 @@
 import '../setup';
 import BN from 'bn.js';
 import StacksAddress from '../../lib/aggregators/stacks-address';
+import { getUnlockedSupply } from '../../lib/core-db-pg/queries';
+
+
+test('get total supply', async () => {
```

Comment:
This test should be in `/test/core-db-pg/queries.test.ts`

Reply:
I just implemented an aggregator for the query and updated the test to use it. Should the test still be moved somewhere else?

Reply:
yeah - now it should go in `/test/aggregators/total-supply.test.ts`